src/js/controller/controller.js

```diff
@@ -577,7 +592,9 @@ Object.assign(Controller.prototype, {
             _actionOnAttach = null;
             checkAutoStartCancelable.cancel();
 
-            _model.set('pauseReason', _getReason(meta));
+            const pauseReason = _getReason(meta);
```

Comment:
What about `playReason` when playback is resumed b/c autoPause?

Reply:
👍